File: Code/Iterator.Java/src/model/Item.java
package model;

/**
 * Business entity. This represent the element we want to store in the data structure.
 */
public class Item {

	private int _payload;

	public int getPayload() {
		return _payload;
	}
	
	public Item(int payload) {
		this._payload = payload;
	}
	
	@Override
	public String toString() {
		return "Element [_payload=" + _payload + "]";
	}

}

File: Code/Iterator.dotNet/Iterator.dotNet/Iterator.dotNet/Model/Item.cs
namespace Iterator.Model
{
    //Business entity. This represent the element we want to store in the data structure.
    internal class Item
    {
        private int _payload;

        public int Payload
        {
            get { return _payload; }
        }

        public Item(int payload)
        {
            _payload = payload;
        }

        override public string ToString()
        {
            return $"Element payload: {_payload}";
        }
    }
}
